lib/charCounter.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+/* eslint-disable */
+
+/**
+ * Function to count how many characters are remaining in text fields
+ *
+ * @param {Object} input - inputs in this plugin's instance and their values as {String}
+ * *
+ * @module charCounter
+ */
+
+module.exports = function charCounter(input) {
```

Comment:
Where are `settings` being passed in so you can get the settings for min and max character length?